fuzzing/cc_deps.bzl

```diff
@@ -43,3 +44,13 @@ def cc_fuzz_test(
         # this attribute must be set.
         testonly = True,
     )
+
+    if "corpus" in kwargs:
+        fuzzing_corpus(
```

Comment:
I'd love to see this passed in to the `fuzzing_launcher` in a later iteration.

So that if I invoke the run target, I start the libfuzzer process with this corpus as a seed corpus by passing it as an arg to the script.

Reply:
I have created #44, and plan to support this feature in the next week if that is okay to you.